tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -164,6 +164,12 @@ Create complex VCH
     Output Should Contain    --container-network-dns=${PUBLIC_NETWORK}:8.8.4.4
     Output Should Contain    --container-network-firewall=${PUBLIC_NETWORK}:outbound
 
+    Output Should Contain    --public-network=${PUBLIC_NETWORK}
+    Output Should Contain    --public-network-gateway=192.168.100.1
+    Output Should Contain    --public-network-ip=192.168.100.22/24
+    Output Should Contain    --dns-server=192.168.110.10
```

Comment:
Just for completeness, can you also make sure the DNS servers show up in the inspect API response bellow?